gramps/gui/views/treemodels/repomodel.py

```diff
@@ -134,7 +134,7 @@ def column_id(self, data):
         return data["gramps_id"]
 
     def column_type(self, data):
-        return str(RepositoryType(data["type"]))
+        return RepositoryType.get_str(data.type)
```

Comment:
I would prefer to see `return str(RepositoryType(data.type))` here, but I can understand why you don't want to instantiate an object.

Reply:
Agreed, just trying to avoid instantiating objects.
The internal implementation is common. `get_str` calls the class method `__get_str`. `__str__` also calls `__get_str`.